Src/main.c

```diff
@@ -303,7 +316,7 @@ alldone(int estatus)
 		if (cutoff)
 		{	sprintf(&pan_runtime[strlen(pan_runtime)], "-u%d ", cutoff);
 		}
-		for (i = 1; i <= PreCnt; i++)
+		for (i = 0; i < PreCnt; i++)
```

Comment:
The above changes to `PreArg` makes this a zero-indexed array, so every loop of this style needs to be updated. However, this variable is only used in this file, which makes this step easy